.github/ISSUE_TEMPLATE/Doc_changes.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Documentation Changes
+about: Create a report to help us improve
+
+---
+
+### Document Change needed
+
+A clear description of what changes are needed with the location of the error that needs to be fixed 
+
+Example:
+In the `master` version of the documentation there is an error in section "13.2.1", the offending line in 17 and can be found in
```

Comment:
Here's a slightly more specific example of a possible error. "Example: There is a broken URL link in the documentation. It is in the `master` version in section 13.2.1 on line 17. The underlying file is `/pecan/book_source/04_advanced_user_guide/01_R_workflow.Rmd`. The correct link should be https://aws.amazon.com/."